src/bp-forums/forums/template.php

```diff
@@ -2552,6 +2554,59 @@ function bbp_get_form_forum_type_dropdown( $args = '' ) {
 	return apply_filters( 'bbp_get_form_forum_type_dropdown', ob_get_clean(), $r );
 }
 
+/**
+ * Checking whether you can update or not forum type.
+ *
+ * Disable "Type" field when the main Forum associated with Group.
+ *
+ * @since bbPress (r3563)
```

Comment:
@asaquzzamanbb why we have `@since bbPress (r3563)` instead of Buddyboss

Reply:
Change `@since BuddyBoss 1.5.9`
https://github.com/buddyboss/buddyboss-platform/pull/2117/commits/da785ba8a0338d2bfbd7d4c829c6d554dfa69f8e, https://github.com/buddyboss/buddyboss-platform/pull/2117/commits/5890058f4ff3e80122785d2eb17b7dd25403a036, https://github.com/buddyboss/buddyboss-platform/pull/2117/commits/114240b580109fa63811374aa0a1f1d1f050cf6f